### DISCUSSION TITLE
Device selection question 2160

### DISCUSSION THREAD

**Discussion by ghost** [gpuweb/gpuweb#2165] [Q&A] [answered]
Follow-up for https://github.com/gpuweb/gpuweb/issues/2160

For a long compute shader, I prefer to use a GPU not connected to a display. Is it possible to select devices using this criteria in Vulkan/DX12/Metal? I don't really know yet. If yes, I'd love to see it in the WebGPU API.

**Comment by kvark** [comment, marked as answer]
That's an interesting request. I think we should discuss this with the group. If I understand correctly, you want that GPU because you don't want to disrupt the work of any rendering, e.g. page contents rendered with GPU, or other work, like games.

**Reply by kainino0x** [reply]
Filed #2167
